README.md

```diff
@@ -1 +1,35 @@
-# tendrl
\ No newline at end of file
+# Tendrl API
+## Installation
+```shell
+$ rbenv install 2.0.0-p647 # Install ruby 2.0.0
+$ gem install bundler
+$ bundle install
+```
+## Development Environment
+Steps to set up the Tendrl API for development.
+
+#### Tendrl Definitions
+The API needs the proper Tendrl definitions yaml file to generate the attributes and actions. You can either download it or use the one from the fixtures to explore the API.
+```shell
+  $ cp spec/fixtures/sds/tendrl_definitions_gluster-3.8.3.yaml config/sds/tendrl_definitions_gluster-3.8.3.yaml 
```

Comment:
Are we maintaining this spec as part of this project as well? Shouln't this be referred from gluster_bridge itself (not sure how though)?